recipes/pkutils/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "pkutils" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 948fe5acb8fd9f315c508756a6420d126400c470e7111408521bb0c7d409c280
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+    - semver
+  run:
+    - python
+    - semver
+
+test:
+  imports:
+    - pkutils
+
+about:
+  home: https://github.com/reubano/pkutils
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE'
```

```suggestion
  license_file: LICENSE
```
did you try this already? Per the [docs](https://conda-forge.org/docs/meta.html#packaging-the-license-manually) it should work.